include/api.php

```diff
@@ -73,31 +73,51 @@ function pll_default_language( $field = 'slug' ) {
 }
 
 /**
- * Among the post and its translations, returns the id of the post which is in the language represented by $lang.
+ * Among the post and its translations, returns the ID of the post which is in the language represented by $lang.
  *
  * @api
  * @since 0.5
+ * @since 3.4 Returns 0 instead of false.
+ * @since 3.4 $lang accepts PLL_Language, string, or int.
  *
- * @param int    $post_id Post id.
- * @param string $lang    Optional language code, defaults to the current language.
- * @return int|false|null Post id of the translation if it exists, false otherwise, null if the current language is not defined yet.
+ * @param int                     $post_id Post ID.
+ * @param PLL_Language|string|int $lang    Optional language (object, slug, or term ID), defaults to the current language.
+ * @return int|null Post ID of the translation if it exists, `0` otherwise, `null` if the language is not defined yet.
+ *
+ * @phpstan-return int<0, max>|null
  */
-function pll_get_post( $post_id, $lang = '' ) {
-	return ( $lang = $lang ? $lang : pll_current_language() ) ? PLL()->model->post->get( $post_id, $lang ) : null;
+function pll_get_post( $post_id, $lang = null ) {
+	$lang = $lang ? $lang : pll_current_language();
+
+	if ( empty( $lang ) ) {
+		return null;
+	}
+
+	return PLL()->model->post->get( $post_id, $lang );
 }
 
 /**
- * Among the term and its translations, returns the id of the term which is in the language represented by $lang.
+ * Among the term and its translations, returns the ID of the term which is in the language represented by $lang.
  *
  * @api
  * @since 0.5
+ * @since 3.4 Returns 0 instead of false.
+ * @since 3.4 $lang accepts PLL_Language, string, or int.
```

Comment:
Same as above.

Reply:
Done in [10ec660](https://github.com/polylang/polylang/pull/1149/commits/10ec660bc67a078dde283dd1c86b5566cee60343).